packages/opentelemetry-exporter-jaeger/src/transform.ts

```diff
@@ -86,6 +86,30 @@ export function spanToThrift(span: ReadableSpan): ThriftSpan {
     });
   }
 
+  /* Add droppedAttributesCount as a tag */
+  if (span.droppedAttributesCount) {
+    tags.push({
+      key: 'otel.dropped_attributes_count',
+      value: toTagValue(span.droppedAttributesCount),
+    });
+  }
+
+  /* Add droppedEventsCount as a tag */
+  if (span.droppedEventsCount) {
+    tags.push({
+      key: 'otel.dropped_events_count',
+      value: toTagValue(span.droppedEventsCount),
+    });
+  }
```

Comment:
Each event also has its own dropped attributes count which I believe should be added where appropriate

Reply:
I'm trying to find where this drop occurs for event attributes. If I had to implement this drop logic, I do not see any `EventLimits` interface. Should the limit thresholds be used from the same `SpanLimits` values?

Reply:
Also, I believe the droppedAttributesCount for events can be added inside `span.event.droppedAttributesCount`, I'm not sure how that will transform into non-OLTP tags, the specification does not talk about this. One way I could imagine is that we add these counts while reporting, but I'm just guessing here.

Reply:
Okay, I referred to the Go repository, I'll add new limit properties inside `SpanLimits`.

Reply:
it's possible we never implemented this limit in JS. if that's the case it would be a separate PR

Reply:
Okay, so I'll split those changes into a separate PR. 
For the other dropped counts, I'll see where I can add them in the exporters.